appcues/src/main/java/com/appcues/Appcues.kt

```diff
@@ -131,6 +133,10 @@ public class Appcues internal constructor(internal val scope: AppcuesScope) {
      * @param properties Optional properties that provide additional context about the user.
      */
     public fun identify(userId: String, properties: Map<String, Any>? = null) {
+        if (pushToken != null && pushToken != storage.pushToken) {
```

Comment:
The idea here is that whenever we identify we first check if the value of pushToken (static) is different than null and also different than the previous storage value, if so we update it before the actual identify call (this will make so session_started will push the new value)